app/views/pupils/schools/_equivalence.html.erb

```diff
@@ -1,14 +1,9 @@
-<div class="row mt-2 mb-2">
-  <div class="col-md-1">
-    <%= image_tag("lightbulb.png", class: "pupil-dashboard") %>
+<div class="row illustration-background">
```

Comment:
This gets sorted properly in the Pupil's update, for now this is removing the lightbulb..